server/src/main/java/com/objectcomputing/checkins/services/role/role_permissions/RolePermissionServicesImpl.java

```diff
@@ -70,6 +76,13 @@ public List<RolePermission> findByRoleId(UUID roleId) {
         return rolePermissionRepository.findByRoleId(roleId);
     }
 
+
+    @Cacheable
+    @Override
+    public List<RolePermission> findByRole(String role) {
+        return rolePermissionRepository.findByRole(role);
+    }
+
     @Override
     public List<Permission> findUserPermissions(@NotBlank UUID id) {
```

Comment:
Any particular reason not to make this method `@Cacheable` as well?

Reply:
Probably not! It's not the one being used in the annotation lookup, but it is used during login.